source/includes/administration/_service_connections.md

```diff
@@ -59,3 +59,92 @@ Attributes | &nbsp;
 `name`<br/>*string* | The name of the service connection
 `type`<br/>*string* | The type of the service connection.
 `status`<br/>*Object* | Status of the service connection. Tells you if the service is up.<br/>*includes*: `lastUpdated`, `reachable`
+
+<!-------------------- GET APIINFO -------------------->
+
+### Retrieve API credentials
+
+`GET /{serviceConnectionId}/apiInfo`
+
+```shell
+# Retrieve your API credentials
+curl -X GET \
+  "https://system.cloudmc-dev.cloudops-devteam.com/rest/services/connections/5b0172cf-00bd-43de-b2e8-9aaf217c8c35/apiInfo?environmentId=ce3b76e6-c4e1-4e11-850d-e31a3055dc8e" \
+  -H "MC-Api-Key: your_api_key" \
```

Comment:
See other examples as to how the URL for the `cURL` is specified.
`curl "https://cloudmc_endpoint/v1/organizations/[id]" -H "MC-Api-Key: your_api_key"`

should use place-holders instead of real URL path params. Also should not include query string unless explained in the API.